### PR TITLE
Add strings and sequences to the pythonic API documentation

### PR DESCRIPTION
**Pull request by yoni206**
This PR adds the documentation for strings and sequences in the pythonic API.

### CODE REVIEW

File: docs/api/python/pythonic/string.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+Strings
+============
+
+
+Basic Sequence Term Builders
```
Comment (alex-ozdemir) [inline docs/api/python/pythonic/string.rst:5]:
```suggestion
Basic String Term Builders
```

File: docs/api/python/pythonic/string.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+.. autofunction:: cvc5.pythonic.Complement
+.. autofunction:: cvc5.pythonic.Plus
+.. autofunction:: cvc5.pythonic.Star
+.. autofunction:: cvc5.pythonic.Loop
+.. autofunction:: cvc5.pythonic.Diff
+
+
+See the following operator overload for string terms:
+
+* is member: :py:meth:`cvc5.pythonic.StringRef.__getitem__`
```
Comment (alex-ozdemir) [inline docs/api/python/pythonic/string.rst:51]: Should this be in a different section, rather than in the regex operators section?

**Review by alex-ozdemir — APPROVED** [review]
LGTM, after two changes.

File: docs/api/python/pythonic/string.rst (hunk 1 above)
Comment (yoni206) [reply docs/api/python/pythonic/string.rst]: fixed.

File: docs/api/python/pythonic/string.rst (hunk 2 above)
Comment (yoni206) [reply docs/api/python/pythonic/string.rst]: moved, and also changed the description (this is not really "is member", which was copied from sets).

**Comment by yoni206** [comment]
@alex-ozdemir thanks for the review. I addressed the comments.

**Pull request merged**